src/components/Table/Table.tsx

```diff
@@ -316,41 +315,72 @@ function Table({
-                        onClick={() => {
-                          if (!column.disableSort) {
-                            toggleSortBy(column.id, column.isSortedDesc === undefined ? false : !column.isSortedDesc)
-                          }
-                        }}
-                      >
-                        {column.render('Header')}
-                      </span>
-                    </Tooltip>
-                    {column.Filter ? column.render('Filter') : null}
-                    {column.isSorted
-                    && (
-                      <div
-                        className='table-sort'
-                        onClick={() => toggleSortBy(column.id, !column.isSortedDesc)}
-                      >
-                        {column.isSortedDesc ? <LongArrow className='rotate180' /> : <LongArrow />}
-                      </div>
-                    )}
-                  </th>
-                ))}
-                {!Utils.isEmpty(rowActions) && (
-                  <th className='table-header table-header-actions'>{EMPTY_STRING}</th>
-                )}
-              </tr>
-            ))}
+            {headerGroups.map((headerGroup) => {
+              const { key, ...headerGroupProps } =
+                headerGroup.getHeaderGroupProps()
+              return (
+                <tr {...headerGroupProps} key={key}>
+                  {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+                  {isExpandable ? (
+                    <th className='table-header header-cell-for-expandable' />
+                  ) : null}
+                  {headerGroup.headers.map((column: ExtendedHeaderGroup) => {
+                    const { key, ...headerProps } = column.getHeaderProps()
+                    return (
+                      <th {...headerProps} key={key} className='table-header'>
+                        <Tooltip data={column.tooltip}>
+                          <span
+                            className={`table-headline ${
+                              column.disableSort ? 'disable-sort' : EMPTY_STRING
+                            }`}
+                            onClick={() => {
+                              if (!column.disableSort) {
+                                toggleSortBy(
+                                  column.id,
+                                  column.isSortedDesc === undefined
+                                    ? false
+                                    : !column.isSortedDesc
+                                )
+                              }
+                            }}
+                          >
+                            {column.render('Header')}
+                          </span>
+                        </Tooltip>
+                        {column.Filter ? column.render('Filter') : null}
+
+                        {!column.disableSort &&
+                          (column.isSorted ||
+                            (!column.isSorted && !column.Header)) && (
```

Comment:
when the header is an empty string, it's impossible to sort the column, so as a workaround I show the sorting button, but with lower opacity